examples/annotation-export/app.py

```diff
@@ -0,0 +1,128 @@
+from pathlib import Path
+
+import matplotlib.dates as mdates
+import matplotlib.pyplot as plt
+import pandas as pd
+import seaborn as sns
+
+from shiny import App, Inputs, Outputs, Session, reactive, render, ui
+from shiny.plotutils import brushed_points
+
+app_ui = ui.page_fluid(
+    ui.panel_title("Plot annotation example"),
+    ui.p(
+        """
+        Select points to annotate them,
+        the plot is drawn with seaborn and all interaction is handled by Shiny.
+        """,
+        {"style": "font-size: larger"},
+    ),
+    ui.row(
+        ui.column(
+            6,
+            ui.output_plot("time_series", brush=ui.brush_opts(direction="x")),
+            ui.output_ui("annotator"),
+        ),
+        ui.column(
+            4,
+            ui.output_data_frame("annotations"),
+        ),
+        ui.column(2, ui.download_button("download", "Download CSV")),
+    ),
+)
+
+
+def server(input: Inputs, output: Outputs, session: Session):
+    annotation_values = reactive.Value(None)
+
+    @reactive.Calc
+    def ts_data():
+        path = Path(__file__).parent / "boulder_temp.csv"
+        dataframe = pd.read_csv(path)
+        dataframe["date"] = pd.to_datetime(dataframe["date"])
+        dataframe["annotation"] = ""
```

Comment:
I think the CSV reading should happen outside of the server function, at the top level of the app, so that it's read only once. Otherwise it'll be read every time any user changes their copy of `annotation_values`.